### ISSUE TITLE
Certificate issue when connecting to secured page

### ISSUE THREAD

**Issue by Ottodix** [TheQwertiest/foo_spider_monkey_panel#120]
Hi,

SMP shows an error message when connecting to a page which have a broken SSL certificate.
```
function callback(){}
this.xmlhttp = ActiveXObject('Microsoft.XMLHTTP');
this.xmlhttp.open("GET", URL);
this.xmlhttp.onreadystatechange = callback;
this.xmlhttp.send();
```
![2020-09-08_204918](https://user-images.githubusercontent.com/19351216/92517149-7e46e880-f216-11ea-948f-9585d8044ac4.jpg)

**Comment by TheQwertiest** [comment]
This is not something SMP can control: the dialog is shown by ActiveX object. I'm not sure if this ActiveX object allows to suppress this dialog window. I will post here though if I find any work-arounds for this.

**Comment by Ottodix** [comment]
Argh... thanks for the answer anyway
This error is quite annoying on my layout, it popup everytime my wifi network got down, because it then connect to a stupid free-and-open-but-require-a-login network. But the right solution would be to fix my internet box anyway, I'm just too lazy to do it ^^
I tried to take a look myself, and didn't find any way so far

**Comment by TheQwertiest** [comment]
Will be most likely fixed with #23, since it will use more modern WinAPI instead of an old ActiveXObject
